index.html

```diff
@@ -19,12 +19,17 @@ <h1>Pick a theme</h1>
       </div>
       <p>Find the tutorial soon on <a href="https://fossheim.io">fossheim.io</a></p>
       </div>
-    </body>
 
   <script>
 
     const pressedButtonSelector = '[data-theme][aria-pressed="true"]';
-    const defaultTheme = 'green';
+    const prefersColorMediaQuery = window.matchMedia("(prefers-color-scheme: dark)");
+    const themes = {
```

Comment:
I'm not very familiar with `window.matchMedia`, what would the return value be when the media query is not supported?